crates/rolldown_common/src/inner_bundler_options/types/optimization.rs

```diff
@@ -64,12 +64,12 @@ pub struct OptimizationOption {
 impl OptimizationOption {
   #[inline]
   pub fn is_inline_const_enabled(&self) -> bool {
-    matches!(&self.inline_const, Some(InlineConstOption::Bool(true) | InlineConstOption::Safe))
+    matches!(&self.inline_const, Some(InlineConstOption::Bool(true) | InlineConstOption::Smart))
   }
 
   #[inline]
   pub fn is_inline_const_safe_mode(&self) -> bool {
```

Comment:
I think we should change this name and this one.
https://github.com/rolldown/rolldown/blob/0fdda0730616c0ba3c2ec2ff73937cb744675574/crates/rolldown/src/stages/link_stage/tree_shaking/include_statements.rs#L91

Reply:
@IWANABETHATGUY `safe_inline_const` is still there.

Reply:
done